packages/api/challenges/speech/recordings.js

```diff
@@ -24,6 +24,19 @@ import { makeWebsocketCall } from '../../communication/websocket';
 const url = challenge => `/challenges/speech/${challenge}/recordings`;
 
 
+/**
+ * Create a new speech recording.
+ *
+ * @param {Object} challengeId - The ID of the challenge to create a recording for.
+ * @param {Object} recording - The recording to create.
+ *
+ * @returns {Promise} - The recording creation promise.
+ */
+export function create(challengeId, recording) {
+  return authorisedRequest('POST', url(challengeId), recording);
```

Comment:
recording as parameter seems not to be tested?

Reply:
What do you mean? It is used in the test

Reply:
Ah, you're right. I was too hasty. the recording object is ```{ id: 'recording' })``` indeed.